easybuild/easyconfigs/p/pkgconfig/pkgconfig-1.5.1-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonBundle'
+
+name = 'pkgconfig'
```

Comment:
@akesandgren We should probably rename this to `pkgconfig-python`, to discriminate wiht `pkg-config`?

Reply:
perhaps, the confusion level increased when it no longer has a -Python suffix.

It will be confusing no matter what we do since the actual pypi package is called pkgconfig.

And if we change this, there is a big bunch of easyconfigs that would need changing ...

Reply:
Only having `pkg-config` and `pkgconfig` with no hint that the latter are Python bindings is *way* more confusing than having `pkg-config` and `pkgconfig-python` imho...

We don't necessarily need to make that change for all existing `pkgconfig` easyconfigs, which have a clear `versionsuffix`...

Another option is to not use `multi_deps` for `pkgconfig` at all, and to keep relying on the `-Python-*` in `versionsuffix` to make it clear these are the Python bindings.

Although I guess that may cause problems when `pkgconfig` is required as a build dependency for something being built with `multi_deps`.

Another option could be to use `-python` as a `versionsuffix`. That avoids a rename, and still makes it clear these are Python bindings?

Reply:
Yeah, that last one feels good, removes confusions and doesn't require changes on older stuff.
Shall we run with that?

Reply:
Makes sense to me.

Reply:
So, lowercase "p"? -> "-python"
or uppercase -> "-Python"

I think lowercase to differentiate from Python with a specific version?

Reply:
@boegel ping

Reply:
@akesandgren lowercase is fine by me